aether-kernel/aether/kernel/api/serializers.py

```diff
@@ -174,7 +174,7 @@ class AttachmentSerializer(DynamicFieldsMixin, serializers.ModelSerializer):
     name = serializers.CharField(allow_null=True, default=None)
     submission_revision = serializers.CharField(allow_null=True, default=None)
 
-    url = serializers.HyperlinkedIdentityField('attachment-detail', read_only=True)
+    url = serializers.CharField(read_only=True, source='attachment_url')
```

Comment:
I think this is a misunderstand, this field it's the API endpoint to update/delete/path this entry, not the file url itself :wink:

Reply:
Ah, I'll take a look.

Reply:
To avoid confusion we can use the `attachment_url` field (also in the `AttachmentSerializerNested` serializer). What do you think?